src/ScheduleFunctions.cpp

```diff
@@ -1748,22 +1817,23 @@ class InjectFunctionRealization : public IRMutator {
         Stmt producer;
         map<string, Expr> replacements;
         vector<pair<string, Expr>> add_lets;
+        map<string, set<string>> aliases;
 
         for (const auto &func_stage : stage_order) {
             const auto &f = func_stage.first;
 
             if (f.has_extern_definition() && (func_stage.second == 0)) {
-                const Stmt &produceDef = Internal::build_extern_produce(env, f, target);
-                producer = inject_stmt(producer, produceDef, LoopLevel::inlined().lock());
+                const Stmt &produce_def = Internal::build_extern_produce(env, f, target);
+                producer = inject_stmt(producer, produce_def, LoopLevel::inlined().lock());
```

Comment:
Were these changes done automatically? Why the churn?

Reply:
This was a drive-by fix to a location variable that didn't match our naming conventions.